testing/MLDB-1984-constant-functions.js

```diff
@@ -14,7 +14,7 @@ function assertEqual(expr, val, msg)
 var query = "SELECT tokenize('a,b,c') AS *";
 var tokQuery = "SELECT tokenize('a,b,c') AS tok";
 
-var analysis = mldb.get("/v1/query", { q: "SELECT static_expression_info(pi())[metadata][isConstant] as isRow", format: 'table', headers: false, rowNames: false });
+var analysis = mldb.get("/v1/query", { q: "SELECT static_expression_info(pi())[\"info\"][isConstant] as isRow", format: 'table', headers: false, rowNames: false });
```

Comment:
I believe it acts as intended? if it finds the keyword "inf" it expects a space, parentesis, comma or such afterwards. The quotes are meant to differentiate between the keywords and the identifiers